torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -11,6 +11,12 @@
 from torch.ao.quantization.qconfig import QConfig
 from torch.ao.quantization.quantize import is_activation_post_process
 
+# Names for observer insert keys
+DEFAULT_TARGET_NODE_KEY = "target_node"
+DEFAULT_OBSERVER_INSERT_KEY = "observer_to_insert"
+DEFAULT_POST_KEY = "is_post_observer"
+DEFAULT_OBS_ARGS_KEY = "observer_args"
```

Comment:
we can remove `DEFAULT` probably since we won't be updating them?

Reply:
we can prefix with something like `DETECTOR_` maybe

Reply:
changed to prefix detector instead, thanks for the suggestion